app/models/publisher.rb

```diff
@@ -355,6 +358,19 @@ def country
     provider_country.to_s.upcase
   end
 
+  def index_to_elasticsearch
+    Search::UserIndexJob.perform_async(id)
+  end
+
+  def index_to_elasticsearch_now
+    return if ENV["ELASTICSEARCH_URL"].blank?
```

Comment:
Should you use `Rails.application.secrets`?

Reply:
https://github.com/brave-intl/publishers/pull/2882/commits/93683426af4a2aeae551d2a68c7ecc0bb9b023db